tests/phpt/expanders/composition_tokenstream_expander.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Uses a custom fully qualified expansion function --pretty-print
+--FILE--
+<?php
+
+$(macro) {
+    hello($(token(T_STRING) as matched));
+} >> {
+    $$(\Yay\tests\fixtures\expanders\my_cheers_tokenstream_expander(
+    	$$(\Yay\tests\fixtures\expanders\my_hello_tokenstream_expander(
+    		$(matched)
+    	))
+    ));
+}
+
+hello(Chris);
+
+?>
+--EXPECTF--
+<?php
+
+'Hello, Chris. From TokenStream. Cheers!';
+
+?>
```

Comment:
This one should be passing now, after latest commit fd6deb9c7867dcbc6f17ccb793450d41ebc63c95